src/os_integrator/main.c

```diff
@@ -110,6 +121,23 @@ int main(int argc, char **argv)
         merror_exit(USER_ERROR, user, group);
     }
 
+    init_conf();
+
+    if (!OS_ReadIntegratorOptions(cfg, &integrator_options)) {
+
```

Comment:
Maybe a warning message?

Reply:
The error message is already shown inside this function:
https://github.com/wazuh/wazuh/blob/86ccb6cd55511e5d9727c8a8bfcd0465720e6371/src/os_integrator/config.c#L28

Reply:
Can you remove the if clause?

Reply:
Fixed in https://github.com/wazuh/wazuh/pull/4146/commits/2a91ed532de385e029597fd44fd239c79fca420b.